.changeset/fifty-dots-rest.md

```diff
@@ -0,0 +1,7 @@
+---
+"@justeattakeaway/pie-wrapper-react": minor
+---
+
+[Changed] - Refactored `add-react-wrapper.js` to be executable via npx.
```

Comment:
I think this should be captured in readme somewhere

Reply:
I don't think this is a command we'd expect people to execute manually, it simply allows for the packages to execute it as part of the `build` npm script.

e.g - `"build": "npx build-react-wrapper pie-button && run -T vite build",`

Reply:
I think the README for react-wrapper needs to be updated to document that this is the way we expect packages to call it